### ISSUE TITLE
Purista cli -> add service breaks under Linux due to case sensitive paths

### ISSUE THREAD

**Issue by kostadinnm** [sebastianwessel/purista#156]
**Describe the bug**
When executing `pnpm purista add service` to append a new service into a project, an error is generated.

**To Reproduce**
Which version do you use:

- PURISTA version: 1.9.0 (latest)
- node ~~/bun/deno~~ version: v20.11.0

**Expected behavior**
A new service scaffold needs to be added.

**Log output**
```
✖  ++ ENOENT: no such file or directory, stat '/home/<my-user>/Projects/purista-demos/users/node_modules/.pnpm/@purista+cli@1.9.0/node_modules/@purista/cli/blueprint/src/service/ServiceEvent.enum.ts.hbs'
✖  ++ src/service/{{camelCase name}}/readme.md Aborted due to previous action failure
✖  ++ src/service/{{camelCase name}}/general{{properCase name}}ServiceInfo.ts Aborted due to previous action failure
✖  ++ src/service/{{camelCase name}}/v{{version}}/readme.md Aborted due to previous action failure
✖  ++ src/service/{{camelCase name}}/v{{version}}/{{camelCase name}}V{{version}}Service.ts Aborted due to previous action failure
✖  ++ src/service/{{camelCase name}}/v{{version}}/{{camelCase name}}V{{version}}Service.test.ts Aborted due to previous action failure
✖  ++ src/service/{{camelCase name}}/v{{version}}/{{camelCase name}}V{{version}}ServiceBuilder.ts Aborted due to previous action failure
✖  ++ src/service/{{camelCase name}}/v{{version}}/index.ts Aborted due to previous action failure
✖  ++ src/service/{{camelCase name}}/v{{version}}/{{camelCase name}}ServiceConfig.ts Aborted due to previous action failure
✖  -> Aborted due to previous action failure
✖  -> Aborted due to previous action failure
 ELIFECYCLE  Command failed with exit code 1
```

**Screenshots**
![2024-01-26_17-58](https://github.com/sebastianwessel/purista/assets/18436405/d472c430-ffe3-41cd-b042-763c7823cf61)

**Additional context**
If I copy `serviceEvent.enum.ts.hbs` to `ServiceEvent.enum.ts.hbs` it passes to create all the `.ts` and `.md` files.

**Comment by kostadinnm** [comment]
I just tried it with standard npm and it's the same thing. There's a commit of yours from the end of March, which is supposed to have both the files, but it seems that node package managers aren't respecting it. I guess, it should just be left with the camel case version of the file name?! Windows should pick it just right.

**Comment by sebastianwessel** [comment]
Hi @kostadinnm,

thanks for your feedback.
There was a copy-paste issue in the CLI source. This should be fixed in the recent published version 1.9.1.
It's published a few minutes ago.

Can you try it out, and come back if you have any issues?

Thanks for trying out PURISTA and your feedback.
